Lib/test/test_free_threading/__init__.py

```diff
@@ -1,7 +1,9 @@
 import os
 
 from test import support
+from unittest import skipUnless
 
 
+@skipUnless(support.Py_GIL_DISABLED, "GIL enabled")
```

Comment:
I have a preference for the more explicit:

```py
if not support.Py_GIL_DISABLED:
    raise unittest.SkipTest("GIL enabled")
```

At the module level.

Reply:
It's not supposed to raise an error. Can I use `unittest.main()`?